indexing-service/src/main/java/io/druid/indexing/overlord/autoscaling/SimpleWorkerProvisioningStrategy.java

```diff
@@ -118,9 +119,9 @@ public synchronized boolean doProvision()
       Collection<? extends TaskRunnerWorkItem> pendingTasks = runner.getPendingTasks();
       Collection<ImmutableWorkerInfo> workers = runner.getWorkers();
       boolean didProvision = false;
-      final WorkerBehaviorConfig workerConfig = workerConfigRef.get();
-      if (workerConfig == null || workerConfig.getAutoScaler() == null) {
-        log.error("No workerConfig available, cannot provision new workers.");
+      final WorkerBehaviorConfig workerConfig =
+          PendingTaskBasedWorkerProvisioningStrategy.getWorkerBehaviorConfig(workerConfigRef, "provision", log);
```

Comment:
should `getWorkerBehaviorConfig` be somewhere else then?

Reply:
What do you mean